app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/topsites/TopSiteItemViewHolder.kt

```diff
@@ -46,10 +48,22 @@ class TopSiteItemViewHolder(
     fun bind(topSite: TopSite) {
         this.topSite = topSite
         view.top_site_title.text = topSite.title
-        view.context.components.core.icons.loadIntoView(view.favicon_image, topSite.url)
+        when {
+            topSite.url == SupportUtils.POCKET_TRENDING_URL -> {
+                view.favicon_image.setImageDrawable(view.context.getDrawable(R.drawable.ic_pocket))
+            }
+            topSite.url.toUri().host?.endsWith(WIKIPEDIA_HOST) == true -> {
+                view.context.components.core.icons.loadIntoView(view.favicon_image, WIKIPEDIA_TOP_URL)
+            }
+            else -> {
+                view.context.components.core.icons.loadIntoView(view.favicon_image, topSite.url)
+            }
+        }
     }
 
     companion object {
+        const val WIKIPEDIA_HOST = "wikipedia.org"
+        const val WIKIPEDIA_TOP_URL = "https://wikipedia.org"
```

Comment:
Workaround because the localized wikipedia URLs don't provide the favicon correctly. I can add a comment if we think it's confusing

Reply:
Let's create a follow-up issue and comment and throw a link to it here 👍